commands/src/api/webrtc.rs

```diff
@@ -0,0 +1,126 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+/// WebRTC client -> server requests
+pub enum Request {
+    /// Get offer from streamer
+    GetOffer(GetOfferRequest),
+    /// Set offer answer from client
+    SetAnswer(SetAnswerRequest),
+    /// Add client's ICE candidate
+    AddCandidate(AddCandidateRequest),
+    /// Get remote ICE candidates from streamer
+    GetCandidates(GetCandidatesRequest),
+    /// Force hang up (close client's streaming session)
+    HangUp(HangUpRequest),
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+/// Request offer form the streamer
+pub struct GetOfferRequest {
+    /// Stream UUID
+    pub stream_id: String,
+}
+
+/// ICE server entry
+/// Refer to https://developer.mozilla.org/en-US/docs/Web/API/RTCIceServer
+#[derive(Serialize, Deserialize, Debug, Default)]
+pub struct IceServer {
+    /// ICE servers URL's array. Note that payload is not an URI, it's format is like "stun:stun.l.google.com:19302"  
+    pub urls: Vec<String>,
+    /// Credential (TURN only)
+    pub credential: Option<String>,
+    /// Username (TURN only)
+    pub username: Option<String>,
+}
+
+#[derive(Serialize, Deserialize, Debug)]
+/// Response on GetOfferRequest
+pub struct GetOfferResponse {
+    /// Peer id identifier, should be user in all future calls
+    peer_id: String,
+    /// ICE servers settings for peer connection construction
+    #[serde(rename = "iceServers")]
+    ice_servers: Vec<IceServer>,
+    /// SDP payload
+    pub sdp: String,
+    /// SDP type
+    pub r#type: String,
```

Comment:
Should we maybe rename this and put `#[serde(rename = "type")]` on it so we don't have to use raw identifier syntax?

Reply:
There was several places with raw identifier across another repos. This one is from here:
https://github.com/lumeohq/cloud-services/blob/master/api/src/api/handlers/streams/webrtc.rs#L23

Reply:
Yeah, I've seen it before. I don't like it much personally, but if you think we should keep it, feel free to just resolve this thread.

Reply:
So, this field reflects this structure as is. Will keep it for now, may be we need some convention for this.
https://developer.mozilla.org/en-US/docs/Web/API/RTCSessionDescription